### PR TITLE
update hpo-config for the templates

### PR DESCRIPTION
**Pull request by yunchu**
updated some hp_config.yaml to change hp search space options.

### CODE REVIEW

**Comment by yunchu** [comment]
run ote-test

File: external/deep-object-reid/configs/ote_custom_classification/efficientnet_v2_s/hpo_config.yaml [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-search_algorithm: smbo
-early_stop: median_stop
+search_algorithm: asha
 hp_space:
   learning_parameters.learning_rate:
-    param_type: quniform
+    param_type: qloguniform
     range:
-      - 0.001
+      - 0.0001
```
Comment (eunwoosh) [inline external/deep-object-reid/configs/ote_custom_classification/efficientnet_v2_s/hpo_config.yaml]: Why value isn't chosen from asha_base, 1, and 2?

File: external/deep-object-reid/configs/ote_custom_classification/mobilenet_v3_small/hpo_config.yaml [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 hp_space:
   learning_parameters.learning_rate:
-    param_type: quniform
-    range: 
-      - 0.005
-      - 0.029
-      - 0.001
+    param_type: qloguniform
+    range:
+      - 0.0014
```
Comment (eunwoosh) [inline external/deep-object-reid/configs/ote_custom_classification/mobilenet_v3_small/hpo_config.yaml]: I think this value is from EfficientNet hp search space candidates.

File: external/mmdetection/configs/custom-object-detection/gen3_mobilenetV2_SSD/hpo_config.yaml [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-search_algorithm: smbo
+search_algorithm: asha
 early_stop: None
 hp_space:
   learning_parameters.learning_rate:
-    param_type: quniform
+    param_type: qloguniform
     range:
       - 0.001
       - 0.1
```
Comment (eunwoosh) [inline external/mmdetection/configs/custom-object-detection/gen3_mobilenetV2_SSD/hpo_config.yaml:9]: if you choose base as hp search space, modifying batch size to base is also needed.

**Review by eunwoosh — DISMISSED** [review]

**Review by harimkang — DISMISSED** [review]
LGTM,
But Is it better to match the values of hpo_config with the values of this PR for MPA?
@goodsong81 @JihwanEom @jaegukhyun @chuneuny-emily

**Comment by yunchu** [comment]
run ote-test

**Comment by yunchu** [comment]
tested report: http://validationreports.sclab.intel.com:8006/reports/build_number_report?test_session_build_number=pr-ote-test-438&environment=iotg-dev-workstation-08

**Comment by yunchu** [comment]
> LGTM, But Is it better to match the values of hpo_config with the values of this PR for MPA? @goodsong81 @JihwanEom @jaegukhyun @chuneuny-emily

I don't think the hpo_config is coupled with specific task so we don't need to update hp_config for the MPA.

**Review by eunwoosh — APPROVED** [review]

**Review by harimkang — APPROVED** [review]

**Pull request merged**